packages/syntax-floating-components/src/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,160 @@
+import { useState, useRef, useEffect } from "react";
+import { StoryObj, Meta } from "@storybook/react";
+import Typography from "../../../syntax-core/src/Typography/Typography";
+import { Tooltip, TooltipContent, TooltipTrigger } from "./Tooltip";
+import image from "../../../../apps/storybook/assets/images/info-icon.svg";
+
+export default {
+  title: "Floating-Components/Tooltip",
+  component: Tooltip,
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/p7LKna9JMU0JEkcKamzs53/%F0%9F%93%90-Syntax?node-id=1007-4109&t=1jAyqXTkYBP57oZL-0",
+    },
+  },
+  argTypes: {
+    initialOpen: {
+      control: { type: "radio" },
+      defaultValue: true,
+      options: [false, true],
+    },
+    open: {
+      table: { disable: true },
+      description:
+        "Value of the 'open' state when controlled (disabled for story)",
+    },
+    onOpenChange: {
+      table: { disable: true },
+      description:
+        "Function to change the value of 'open' when tooltip is interacted (diabled for story)",
+    },
+    placement: {
+      control: { type: "select" },
+      defaultValue: "right",
+      options: [
+        "top",
+        "top-start",
+        "top-end",
+        "right",
+        "right-start",
+        "right-end",
+        "bottom",
+        "bottom-start",
+        "bottom-end",
+        "left",
+        "left-start",
+        "left-end",
+      ],
+      table: {
+        defaultValue: { summary: "right" },
+      },
+    },
+    strategy: {
+      control: { type: "select" },
+      options: ["absolute", "fixed"],
+      defaultValue: "absolute",
+      table: {
+        defaultValue: { summary: "absolute" },
+      },
+    },
+    children: {
+      control: { type: "text" },
+      description: "The string value to show on the tooltip content",
+      defaultValue: "This is a button",
+      table: {
+        type: { summary: "string" },
+        defaultValue: { summary: "This is a button" },
+      },
+    },
+  },
+  tags: ["autodocs"],
+} as Meta<typeof Tooltip>;
+
+export const Default: StoryObj<typeof Tooltip> = {
+  args: {
+    delay: 0,
+    placement: "right",
+    initialOpen: true,
+    strategy: "absolute",
+    children: "This is a book",
```

Comment:
technically,`children` should be an react element but I had to pass it in as a string in storybook b/c it complained if I tried to pass in an arg that was a part of `Tooltip`